services/web/nginx.dev.conf

```diff
@@ -12,6 +12,9 @@ server {
     # giving information about what we should expect to find in the request body
     include mime.types;
 
+    auth_basic "Restricted Access";
+    auth_basic_user_file /conf/basic-auth/nginx/.htpasswd;
```

Comment:
So the `.htpasswd` file is created and stored on the Dev machine at `/etc/nginx/.htpasswd`, right? The content of this file is the hashed password. If yes to both of these, merge.